lib/src/subscriber.dart

```diff
@@ -0,0 +1,564 @@
+import 'dart:async';
+import 'dart:math';
+
+import 'package:collection/collection.dart';
+import 'package:sip_ua/sip_ua.dart';
```

Comment:
Same here, I prefer relative paths, it looks like less code, :)

Reply:
Is this better?

Reply:
This makes it easier to distinguish between internal imports or external package imports, I think.

Reply:
Makes sense, I am farily new to dart, and as such I dont know all the finder details.